app/src/main/java/ca/llamabagel/transpo/settings/data/AppSettings.kt

```diff
@@ -0,0 +1,18 @@
+package ca.llamabagel.transpo.settings.data
+
+import io.github.dellisd.quicksave.SettingsProvider
+import io.github.dellisd.quicksave.booleanSetting
+import javax.inject.Inject
+import javax.inject.Singleton
+
+@Singleton
+class AppSettings @Inject constructor(provider: SettingsProvider) : SettingsProvider by provider {
+    val groupByRoute by booleanSetting {
+        title = "Group results by Route"
+        caption = "Group trips by their Route number and destination"
```

Comment:
we should add string resources for these

Reply:
Currently unsupported by the library, but noted (for addition to the library).

Reply:
I'm not sure I want to inject a `StringProvider` into there when the library could simply handle accepting resource ids directly. Injecting `StringProvider` will be the fall back.